### PR TITLE
T 5 list of news

### PR DESCRIPTION
**Pull request by girlpluscomputer**
Styles will be changed.

### CODE REVIEW

File: src/views/Home/Home.js [hunk 1]
```diff
@@ -1,7 +1,84 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Navbar, Content } from './style';
```
Comment (arathjz) [inline src/views/Home/Home.js]: This is not a third party dependencie

File: src/views/Home/Home.js [hunk 2]
```diff
@@ -1,7 +1,84 @@
...
+import BaselineAccountCircle from 'react-md-icon/dist/BaselineAccountCircle';
+
+import { getStories, getStory } from './api';
```
Comment (arathjz) [inline src/views/Home/Home.js]: This should be at the bottom of the imports, it should be like

```
import React, { useEffect, useState } from 'react';
import { Link } from 'react-router-dom';
import BaselineAccountCircle from 'react-md-icon/dist/BaselineAccountCircle';

import CardStorie from './components/card';
import Loader from '../../components/Loader';
import logo from '../../static/logo-white.png';
import CardStorie from './components/card';
import { Container, Navbar, Content } from './style';
import { getStories, getStory } from './api';

```

File: src/views/Home/components/card/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Card from './style';
+import BaselineStarBorder from 'react-md-icon/dist/BaselineStarBorder';
+
+const CardStorie = ({ story, formatDate }) => {
+  const { time, title, by } = story;
```
Comment (arathjz) [inline src/views/Home/components/card/index.js]: Add a line break here please

File: src/views/Home/components/card/style.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
```
Comment (arathjz) [inline src/views/Home/components/card/style.js]: Add a line break here

File: src/views/Home/components/card/style.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import theme from '../../../../theme';
```
Comment (arathjz) [inline src/views/Home/components/card/style.js]: Remove this line, you should not access theme this way

File: src/views/Home/components/card/style.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    }
+  }
+
+  .date {
+    font-size: ${theme.font.sm};
+    margin: 0;
+  }
+
+  .title {
+    font-size: ${theme.font.md};
```
Comment (arathjz) [inline src/views/Home/components/card/style.js]: The way you should access to this is like

`font-size: ${props => props.theme.font.md};`

File: src/views/Login/Login.js [hunk 7]
```diff
@@ -1,7 +1,92 @@
...
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import * as firebase from 'firebase/app';
+import 'firebase/auth';
+import toast from 'toast';
 
-const Login = () => {
-  return <div>Login</div>;
+import { Button, Container, Loader } from '../../components';
+import Card from './style';
```
Comment (arathjz) [inline src/views/Login/Login.js]: This should be at the bottom of the imports due the folder roots is not the same, example

```
import { Button, Container, Loader } from '../../components';
import logo from '../../../src/static/logo-primary.png';
import Card from './style';

```

Theme should not be imported here, you should not import theme anywhere, only where the ThemeProvider is

File: src/views/Login/Login.js [hunk 8]
```diff
@@ -1,7 +1,92 @@
...
+    const {
+      target: { name, value },
+    } = e;
+
+    setData({ ...data, [name]: value });
+  };
+
+  return (
+    <Container>
+      <Card theme={theme}>
```
Comment (arathjz) [inline src/views/Login/Login.js]: Card already has theme in its props due is inside the ThemeProvider, theme should not be pass in props

File: src/views/Login/Login.js [hunk 9]
```diff
@@ -1,7 +1,92 @@
...
+            placeholder="Password"
+          />
+          <div className="button-container">
+            <div className="some-text">
+              Don't have an account? <Link to="/register">Register</Link>
+            </div>
+            {loading ? (
+              <Loader />
+            ) : (
+              <Button type="submit" theme={theme}>
```
Comment (arathjz) [inline src/views/Login/Login.js]: Same here

File: src/views/Login/style.js [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+  justify-content: center;
+
+  .image-container {
+    display: flex;
+    justify-content: center;
+    background-color: #fff;
+  }
+
+  .title {
+    font-size: ${props => props.theme.font.xl};
```
Comment (arathjz) [inline src/views/Login/style.js]: Exactly, this is already in props from ThemeProvider

File: src/views/Register/Register.js [hunk 11]
```diff
@@ -1,30 +1,30 @@
...
 import * as firebase from 'firebase/app';
-import "firebase/auth";
+import 'firebase/auth';
 
 import toast from 'toast';
 
 import { Button, Container, Loader } from '../../components';
-import Card from './style'
-import logo from '../../../src/static/logo.png';
+import Card from './style';
```
Comment (arathjz) [inline src/views/Register/Register.js]: Order matters

```
import logo from '../../../src/static/logo-primary.png';
import Card from './style';
```

File: src/views/Register/Register.js [hunk 12]
```diff
@@ -73,14 +73,13 @@ const Register = ({setIsAuthenticated}) => {
...
-                : (
-                  <Button type="submit" theme={theme}>
-                    Continue
-                  </Button>
-                )
-              }
+            {loading ? (
+              <Loader />
+            ) : (
+              <Button type="submit" theme={theme}>
```
Comment (arathjz) [inline src/views/Register/Register.js]: Theme should not be imported and passed by props, its already in all styled components props

File: src/views/Home/Home.js (hunk 1 above)
Comment (girlpluscomputer) [reply src/views/Home/Home.js]: Changed

**Pull request merged**